clickhouse/columns/string.cpp

```diff
@@ -190,6 +190,13 @@ ColumnString::ColumnString(std::vector<std::string>&& data)
 ColumnString::~ColumnString()
 {}
 
+void ColumnString::Reserve(size_t new_cap)
+{
+    items_.reserve(new_cap);
+    // 16 is arbitrary number, assumption that string values are about ~256 bytes long.
+    blocks_.reserve(std::max<size_t>(1, new_cap / 16));
```

Comment:
Why is it different from https://github.com/ClickHouse/clickhouse-cpp/blob/6a8729c5e572146981387275aec62d6b8c0e15ac/clickhouse/columns/string.cpp#L164 ?

Reply:
Usually, 40bytes is a little short, especially for other language (no english). 
I make both them same now.